examples/build_all.py

```diff
@@ -1,29 +1,46 @@
 #!/usr/bin/python3
 
+import os
+
 from pathlib import Path
 from subprocess import PIPE, run
 
+# NOTE: boost V1.67 is to old! CK
 examples = [
-  x for x in Path(__file__).parent.iterdir() if x.is_dir() and (x / 'CMakeLists.txt').exists()
+    x for x in Path(__file__).parent.iterdir() if x.is_dir() and (x / 'CMakeLists.txt').exists() and (not x.name in ['boost', 'old-gtest'])
 ]
 
 assert(len(examples) > 0)
 
+
 def runCommand(command):
-  print('- %s' % command)
-  result = run(command, stdout=PIPE, stderr=PIPE, universal_newlines=True, shell=True)
-  if result.returncode != 0:
-    print("error while running '%s':\n" % command, '  ' + str(result.stderr).replace('\n','\n  '))
-    exit(result.returncode)
-  return result.stdout
+    print('- %s' % command)
+    result = run(command, stdout=PIPE, stderr=PIPE, universal_newlines=True, shell=True)
+    if result.returncode != 0:
+        print("error while running '%s':\n" %
+              command, '  ' + str(result.stderr).replace('\n', '\n  '))
+        exit(result.returncode)
+    return result.stdout
```

Comment:
This project uses 2-space indentation throughout for consistency. Could you reformat the python to use 2-space indentation as before?

Reply:
yes, I used autopep8 with it defaults